src/collection.js

```diff
@@ -79,13 +79,18 @@ export default class Collection {
    *
    * @param  {Object} [options={}]      The options object.
    * @param  {Object} [options.headers] The headers object option.
+   * @param  {String} [options.since]   The since ETag option.
    * @return {Promise<Number, Error>}
    */
   getTotalRecords(options={}) {
     const { headers } = this._collOptions(options);
```

Comment:
actually, the `const { since, ...otherOptions } = options` instruction breaks the firefox build:
https://travis-ci.org/Kinto/kinto-http.js/jobs/163077307#L959

Maybe this issue: https://github.com/substack/module-deps/issues/93